### ISSUE TITLE
References to old name "content-type" in documentation and UI

### ISSUE THREAD

**Issue by chinhdo** [strapi/strapi#5555]
"content type" is now "collection type" but the UI and documentation still has many references to "content types".

**Steps to reproduce**

1. Go to [Quick Start Guide](https://strapi.io/documentation/3.0.0-beta.x/getting-started/quick-start.html#_2-create-an-administrator-user) and scroll down to "3. Create a Restaurant Content Type"
2. The first step says 'Click the "+ Create new content-type" link'. Should say "+ 'Create new collection type" link'.

**Steps to reproduce**
1. Log into the Admin site
2. On the left sidebar, under PLUGINS, the "Content-Types Builder" should say "Collection-Types Builder"

**Comment by alexandrebodin** [comment]
Hi @chinhdo, Thanks your for reporting this :)

The getting started should be updated to clarify ;) Feel free to open a PR. Notifying @lauriejim who handles this :)

To clarify: The name Content Type is still valid in the context of the Content types builder.  A `Collection Type` is a `Content Type`. and a `Single Type` is also a `Content Type` ;)

**Comment by derrickmehaffy** [comment]
I've spoken with @lauriejim the past few days on this relating to parts of the documentation as well as many example images on the website as well.

**Comment by chinhdo** [comment]
@alexandrebodin Would be happy to open a PR for Getting Started Guide if this isn't already being worked on by someone. In Slack it sounds like someone else may be on it already.

**Comment by alexandrebodin** [comment]
@lauriejim might give you the answer :)

**Comment by derrickmehaffy** [comment]
@alexandrebodin Was going to review the docs this week myself. @chinhdo if you want to take a look through as well that would be great :wink: had a lot going on over the weekend that I couldn't get to it sooner.

**Comment by chinhdo** [comment]
@alexandrebodin @lauriejim Sure, I will create a PR to fix incorrect reference to '+ Create new "content-type" link' and other similar references in the Quick Start Guide.

Also I think users can benefit from having some documentation explaining core concepts like ```Content Type```, ```Single Type```, and ```Collection Type``` - why they exist, how they are used, etc. I tried to search the docs but couldn't find much.

**Comment by chinhdo** [comment]
Found this very good blog post by @yvesdo that talks about "Collection Type" and "Single Type". [Announcing beta.19 - Single Types & UID field](https://strapi.io/blog/beta-19-single-types-uid-field)

**Comment by chinhdo** [comment]
[Pull Request](https://github.com/strapi/strapi/pull/5635) created to fix incorrect/outdated references to renamed menu items in Quick Start Guide/Tutorial.  There needs to be a review of the docs. This PR is intended just to fix just the Quick Start Guide/Tutorial to prevent confusing/frustrating people brand new to Strapi.

**Comment by adriatic** [comment]
I just received the response from Christian Guevara posted in slack:

![image](https://user-images.githubusercontent.com/2712405/78092756-96305b80-739e-11ea-98a9-7e1bb198d8e6.png)

(he replied to my post that reported the **[same issues](https://github.com/strapi/strapi/issues/5555#issue-585068982)**, so he referred me to this thread.

Just like @chinhdo I wanted to create a PR to address this and wanted to install the **[latest beta 19.4](https://github.com/strapi/strapi/releases/tag/v3.0.0-beta.19.4)** before doing that. 

@chinhdo can you please let me if your PR is written for 19.4 - and if so, I will be happy to apply it to my local source tree and verify your changes.
, 
cc: @derrickmehaffy, @lauriejim

**Comment by chinhdo** [comment]
Hi @adriatic : 

Here's my [PR](https://github.com/strapi/strapi/pull/5635). The source branch "fix/5555-quickStartGuide was created from master and I also merged from master just now.

I see a notification on my PR saying "@alexandrebodin added this to the 3.0.0-beta.20 milestone 2 days ago".

Not sure if that answered your question.

**Comment by adriatic** [comment]
Thank you, @chinhdo. It seems that these corrections are slated for 3.0.0-beta.20 milestone 2, so my task here is to re-verify them and report my findings back to you as the author. I offered to do that because I am going through Quick Start Guide documentation.

**Comment by chinhdo** [comment]
@adriatic Sure if you can help review the changes that would be great. Please note that the focus of the PR is on addressing the obvious outdated references in the Quick Start Guide/Tutorial. The team still needs a wider review of usages of "Content Type", "Collection Type", "Single Type" in the rest of the documentation and in the UI itself.

**Comment by adriatic** [comment]
**`Please note that the focus of the PR is on addressing the obvious outdated references in the Quick Start Guide/Tutorial`**

That is precisely my motivation, driven by my firm belief that the tutorial most new users encounter first simply ought to be free of any errors. I run that tutorial being really excited about Strapi as a concept, and almost gave up, thinking "these guys to not realize that advancing the code functionality without simultaneous additions to the documentation is leading to the company's demise"

Note that it is likely that the video portions of this tutorial would either have be recreated immediately, or hidden until recreated.

**Comment by adriatic** [comment]
#### It seems that I need to go back and learn a lot more about building strapi from sources

I tried to circumvent merging your PR to my local clone of the **`Strapi`** master, after realizing that @lauriejim already did that a **[few hours back](https://github.com/strapi/strapi/commit/0a522b3795a423eb0616ac50a947121e2e3d0257)**.

So, in order to do the verification I offered, I simply cloned the current master branch to my computer and then used the **[CONTRIBUTING](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md)** document to see the Quick Start Guide document in its proper environment. Specifically, I run the **`yarn build`** and then 
```
cd strapi/packages/strapi-admin
yarn develop
```
The app fails to start with:

![image](https://user-images.githubusercontent.com/2712405/78167914-40e75f00-741d-11ea-8723-86598f101245.png).

Obviously I do not know how to really build the app from sources 😢, so my offer to help, resulted in more commotion that it is worth.

**Comment by chinhdo** [comment]
@adriatic For documentation, I think you can skip the app-building part if all you want to do is review the updated docs. You can just look at the documentation markdown files directly.

Can you try this?
- Checkout master
- Navigate to the folder /docs/3.0.0-beta.x/getting-started
- View the files quick-start.md and quick-start-tutorial.md in a code editor like Visual Studio Code. With Visual Studio Code you can also "preview" *.md files so you can see how the files would render on the web.

![image](https://user-images.githubusercontent.com/2593591/78171324-472c0a00-7422-11ea-9271-ee57cdd84183.png)

Chinh

**Comment by adriatic** [comment]
Yes, absolutely - your suggestion is very obvious - and I did not already do that simply because I was worried about any possible "transformations" behind the scene in strapi build process. I will report back here

**Comment by derrickmehaffy** [comment]
> @adriatic For documentation, I think you can skip the app-building part if all you want to do is review the updated docs. You can just look at the documentation markdown files directly.
> 
> Can you try this?
> 
> * Checkout master
> * Navigate to the folder /docs/3.0.0-beta.x/getting-started
> * View the files quick-start.md and quick-start-tutorial.md in a code editor like Visual Studio Code. With Visual Studio Code you can also "preview" *.md files so you can see how the files would render on the web.
> 
> ![image](https://user-images.githubusercontent.com/2593591/78171324-472c0a00-7422-11ea-9271-ee57cdd84183.png)
> 
> Chinh

@chinhdo @adriatic I'll do you both one better.

Clone the mono repo, navigate to docs, and run `yarn` then `yarn dev` and it will spin up a local hot-reloading copy of the docs :wink:

**Comment by adriatic** [comment]
@derrickmehaffy @chinhdo - this is clearly at least `one better`. As I am almost done with terifying the **[second article](https://strapi.io/documentation/3.0.0-beta.x/getting-started/quick-start-tutorial.html#tutorial)**. Once I am done, I will rebuild the docs as @derrickmehaffy suggested, since there is some amount of processing the markdown file, which is not present when following @chinhdo' "method". 

I am slow doing this verification as I do not want to "hide" behind other folks who had a look at that. I am also learning the basics of strapi "visual builder", and making copious notes,

**Comment by adriatic** [comment]
### Status update
I am finding bugs unrelated to this verification tasks - they are clearly older than switch from **`Content`** to **`Collection`**. 

Since I found some errors still in the current master, I would think that @chinhdo might decide to recreate his original PR, instead of "patching the patch" (this is of course a suggestion as I do not know your operating procedure). Recreating the original OR will also allow verification of my findings in this thread.

More tomorrow ...

**Comment by chinhdo** [comment]
Hi @adriatic : Thanks for reviewing my PR. Since it's already been merged to master and approved for the next version, can you use a new PR for any further corrections/changes?

**Comment by adriatic** [comment]
Hello, @chinhdo . Good thinking, I will gladly to this. Is there a document that describes the "standard Strapi PR creation? (I never did one for the monorepo, so I do not want to step on someone's toe 😄)

cc: @derrickmehaffy

**Comment by chinhdo** [comment]
@adriatic You'll want to look at the [Contributing Guide](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md) . Basically you fork the repo, create a new branch, and when ready, create a PR from your branch back to master. Chinh

**Comment by derrickmehaffy** [comment]
Exactly as described ;)

**Comment by adriatic** [comment]
@chinhdo - so no differences due to monorepo, it seems. Thanks, taking this over,
